CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Changelog
 
 - Updated Electron and related packages ([PR 2206](https://github.com/input-output-hk/daedalus/pull/2206))
 - Fixed some Japanese translations for the external currencies ([PR 2667](https://github.com/input-output-hk/daedalus/pull/2667))
+- Fix crashes when Yubikey is connected ([PR 2673](https://github.com/input-output-hk/daedalus/pull/2673))
```

Comment:
@danielmain please move this line to the top of "Fixes" section and make sure to use past tense - "Fix crashes..." -> "Fixed crashes..." 🙏

Reply:
done

Reply:
Thanks 😊